cmd/subnetcmd/addValidator.go

```diff
@@ -261,7 +261,9 @@ func promptStart() (time.Time, error) {
 }
 
 func promptNodeID() (ids.NodeID, error) {
-	txt := "What is the NodeID of the validator you'd like to whitelist?"
+	txt := "What is the NodeID of the validator you'd like to whitelist?\n" +
```

Comment:
I'm not sure if this will work as is. The prompt library doesn't handle multiline strings well.